packages/components/src/palette-edit/index.js

```diff
@@ -321,6 +307,17 @@ export default function PaletteEdit( {
 							{ ( { onClose } ) => (
 								<>
 									<NavigableMenu role="menu">
+										<Button
+											variant="tertiary"
+											disabled={ isEditing }
+											onClick={ () => {
+												setIsEditing( true );
+												onClose();
+											} }
+											className="edit-palette-menu-button"
+										>
```

Comment:
Would it be possible to make this CSS name more consistent with others in the component, e.g., `components-palette-edit__menu-button`

Reply:
good point - have made this change